scripts/gatsby-source-iiif/indexSearch.js

```diff
@@ -14,14 +14,13 @@ if (appConfig === 'local' || process.env.TRAVIS_RUN) {
 
 const directory = process.argv.slice(2)[0]
 require('dotenv').config({
-  path: path.join(directory, '.env.production'),
+  path: path.join(directory, '.env.development'),
```

Comment:
Do we need logic here to detect environment?

Reply:
in this case no because this is not run in development at the moment.  But this is an issue if we want to run a local index.  I am currently tempted to change all the devs to run of the libnd test index so it that gets built when test site is built.

Reply:
👍